### PR TITLE
Update to fabric proxy model doc

### PR DESCRIPTION
**Pull request by Ram-srini**
Add instructions to setup the Avalon fabric proxy model on multi node network.
To address the git issue - https://github.com/hyperledger/avalon/issues/627

Signed-off-by: Ramakrishna Srinivasamurthy <ramakrishna.srinivasamurthy@intel.com>

### CODE REVIEW

File: docs/TestingFabricProxyModel.rst [hunk 1]
```diff
@@ -92,6 +94,57 @@ To run Fabric with Hyperledger Avalon, follow these steps:
 
        ./scripts/start_fabric.sh -d
 
+Setup Hyperledger Avalon Fabric Proxy Model on multiple machine
+----------------------------------------------------------------
+Multinode Hyperledger Avalon fabric proxy model setup requires certain modifications to existing
+docker files. For example to setup the fabric blockchain network on a separate machine and avalon on a separate machine
```
Comment (danintel) [inline docs/TestingFabricProxyModel.rst:100]: Change avalon to Avalon here on line 100 and on line 105, 114, 126.

File: docs/TestingFabricProxyModel.rst [hunk 2]
```diff
@@ -92,6 +94,57 @@ To run Fabric with Hyperledger Avalon, follow these steps:
...
+
+   .. code:: sh
+        scp -r ~/mywork <user_name>@<IP1>:~/
+
+3. If you choose the customized setup other than the start_fabric.sh (minifab) then create the network.json as in (https://github.com/hyperledger/avalon/blob/master/sdk/avalon_sdk/connector/blockchains/fabric/network.json)
+
+4. If these 2 matchines are behind the corporate network please update environment variables `no_proxy` and `NO_PROXY`
+   with the IP1 on machine2 as mentioned above.
+
+5. Remove the network tag from the docker compose file - https://github.com/hyperledger/avalon/blob/master/docker/compose/avalon-fabric.yaml#L55
```
Comment (danintel) [inline docs/TestingFabricProxyModel.rst:124]: Change docker to Docker here on line 124 and line 136

File: docs/TestingFabricProxyModel.rst (hunk 1 above)
Comment (Ram-srini) [reply docs/TestingFabricProxyModel.rst]: Done

File: docs/TestingFabricProxyModel.rst (hunk 2 above)
Comment (Ram-srini) [reply docs/TestingFabricProxyModel.rst]: Done

**Pull request merged**